requirements.txt

```diff
@@ -1,2 +1,5 @@
 discord.py==2.1.0
 requests==2.28.1
+psycopg==3.1.8
```

Comment:
you sure we need both the basic lib & binary? have you tried just lib no bin & just bin no lib

Reply:
The correct installation based off the docs https://www.psycopg.org/psycopg3/docs/basic/install.html should be
`psycopg[binary]==3.1.8`
Resolved

Reply:
lgtm